python-checks/src/test/resources/checks/argumentType.py

```diff
@@ -86,8 +88,8 @@ def builtin_functions():
   number = 42
   number.__add__("27") # Noncompliant
   number.__add__(unexpected) # Noncompliant
-  number.__add__(x = unexpected) # Noncompliant {{Change this argument; Function "__add__" expects a different type}}
-#                ^^^^^^^^^^^^^^
+  number.__add__(x = unexpected) # FN Noncompliant {{Change this argument; Function "__add__" expects a different type}}
```

Comment:
Here what happens is that we don't recognize `x` as being the name of the parameter (it's now empty).